woocommerce/wc-admin
# Add pre-launch checklist note on task list completion

Fixes #4211 

Adds a note when the task list is completed or if 3 tasks have been completed within 10 days.

### Screenshots
<img width="715" alt="Screen Shot 2020-06-09 at 3 47 37 PM" src="https://user-images.githubusercontent.com/10561050/84149909-d4ff2380-aa69-11ea-9d71-512663081da5.png">


### Detailed test instructions:

1. On a newer site ( < 10 days old ) with < 3 tasks completed, run the `wc_admin_daily` cron event.
1. Make sure the note is not added.
1. Complete at least 3 tasks.
1. Trigger `wc_admin_daily` again and make sure the note is added.
1. Delete the note and modify the `woocommerce_admin_install_timestamp` option to be older than 10 days.
1. Trigger `wc_admin_daily` and make sure the note is not added.
1. Complete all tasks.
1. Trigger `wc_admin_daily` and  make sure the note is added.
